test/Microsoft.Extensions.Configuration.FunctionalTests/ConfigurationTests.cs

```diff
@@ -246,7 +246,7 @@ public void GetDefaultBasePathForSources()
 #if DNXCORE50
             filePath = AppContext.BaseDirectory;
 #else
-            filePath = Path.GetFullPath((string)AppDomain.CurrentDomain.GetData("APP_CONTEXT_BASE_DIRECTORY"));
+            filePath = Path.GetFullPath((string)AppDomain.CurrentDomain.BaseDirectory);
```

Comment:
`APP_CONTEXT_BASE_DIRECTORY` is gone. I changed it here because the test was failing after switching away from dnx